src/info/index_error.c

```diff
@@ -89,8 +92,17 @@ const char *IndexError_LastError(const IndexError *error) {
 }
 
 // Returns the key of the document that caused the error.
-const RedisModuleString *IndexError_LastErrorKey(const IndexError *error) {
-    return error->key;
+RedisModuleString *IndexError_LastErrorKey(const IndexError *error, bool obfuscate) {
+    if (!obfuscate) {
+      return RedisModule_HoldString(RSDummyContext, error->key);
```

Comment:
Why use `HoldString`? Add some documentation please

Reply:
Will add some documentation above the call.
Basically because in the else case we create a string.
Wanted the caller to always free the string he got.